ci/k8s/client/test_p2p_k8s.py

```
@@ -0,0 +1,131 @@
+    assert text, f"Empty completion text from {job_name}: {body}"
+
+
+def test_rdma_transfer_logged(namespace: str, p2p_marker: str) -> None:
+    """Target pod logs must contain the framework's P2P transfer marker.
+
+    Absence means the target loaded weights via a fallback path, not RDMA.
+    """
+    pod = _pod_name(namespace, "mx-target")
+    print(f"\n[mx-target] pod={pod}")
+    result = _kubectl("logs", pod, "-c", "mx-target", "--tail=-1", namespace=namespace)
+    marker_lines = [l for l in result.stdout.splitlines() if any(k in l for k in ("RDMA", "P2P", "transfer"))]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)